test/multifile/synthesized-accessors/materialize-for-set-1/library.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: true
+
+import Foundation
+
+public protocol GeneratesDecimalNumbers {
+  var generatesDecimalNumbers: Bool { get set }
+}
+
+extension NumberFormatter : GeneratesDecimalNumbers {}
```

Comment:
Please use the mock SDK or a handwritten class rather than the system Foundation.